lc0_main.go

```diff
@@ -1064,7 +1065,12 @@ func main() {
 
 	log.SetFlags(log.LstdFlags | log.Lshortfile)
 	if len(*user) == 0 || len(*password) == 0 {
-		*user, *password = readSettings("settings.json")
+		var settingsHost string
+		*user, *password, settingsHost = readSettings("settings.json")
+
+		if (len(settingsHost) != 0) {
+			*localHost = settingsHost
+		}
```

Comment:
Setting localHost from the json file should be independent from the user/password setting.

Reply:
Hrm, Ok.  That would probably require splitting up the readSettings function a bit.  Otherwise, if username and password were passed on the command line but localHost was in settings.json the current logic would cause ask the username/password to be entered again.

Reply:
Or just read them all in temporary variables a bit earlier

Reply:
Done